src/sentry/rules/conditions/active_release.py

```diff
@@ -5,40 +5,26 @@
 from django.utils import timezone
 
 from sentry.eventstore.models import Event
-from sentry.models import (
-    Deploy,
-    Environment,
-    Project,
-    Release,
-    ReleaseEnvironment,
-    ReleaseProjectEnvironment,
-)
+from sentry.models import Deploy, Release, ReleaseEnvironment, ReleaseProjectEnvironment
 from sentry.rules import EventState
 from sentry.rules.conditions.base import EventCondition
-from sentry.search.utils import get_latest_release
 
 
-def _get_latest_release(
-    organization_id: int, environment_id: Optional[int], project: Project
-) -> Optional[Release]:
-    environments = (
-        None if not environment_id else [Environment.objects.filter(id=environment_id).first()]
-    )
-
-    try:
-        latest_release_versions = get_latest_release(
-            [project],
-            environments,
-            organization_id,
+def _get_release(event: Event) -> Optional[Release]:
+    if event:
+        release_version = (
+            event.release
```

Comment:
I think it's safe to just use `event.release`, although not sure if you saw somewhere else that people are doing this?

Reply:
Just being overly defensive; I think they're all aliases for the same thing.